test/unit/integration/body-removal-before-unmount.spec.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react';
+import { render } from '@testing-library/react';
+import { isDragging } from './util/helpers';
+import App from './util/app';
+import { forEachSensor, simpleLift, type Control } from './util/controls';
+import getBodyElement from '../../../src/view/get-body-element';
+
+it('should have any errors when body is changed just before unmount', () => {
+  jest.useFakeTimers();
+  const { unmount } = render(<App />);
+
+  expect(() => {
+    getBodyElement().innerHTML = '';
+    unmount();
+    jest.runOnlyPendingTimers();
+  }).not.toThrow();
+
+  jest.useRealTimers();
+});
+
+forEachSensor((control: Control) => {
+  it('should have any errors when body is changed just before unmount: mid drag', () => {
```

Comment:
💯 to this test.  Much better sorry I should have probably just migrated it here :)

Reply:
All good! I wanted to get yours in and see if i could make it catch every component